eth/gaspricemonitor.go

```diff
@@ -54,6 +56,7 @@ func NewGasPriceMonitor(gpo GasPriceOracle, pollingInterval time.Duration, minGa
 		pollingInterval: pollingInterval,
 		gasPrice:        big.NewInt(0),
 		minGasPrice:     minGasP,
+		maxGasPrice:     maxGasPrice,
```

Comment:
Let's add a metrics recording call for maxGasPrice similar to the one we have for minGasPrice on L52 (will probably need a nil check as well in case maxGasPrice is nil).

Reply:
fixed in 91ceac9